scripts/clean-install.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+
+
+function cleanup() {
+  echo "Cleaning up artifacts"
+  rm -rf ./clean
+  rm -rf ./firebase-tools-*.tgz
+  echo "Artifacts deleted"
+}
+
+trap cleanup EXIT
+
+echo "Installing clean-publish"
+npm install -g clean-publish
+echo "Installed clean-publish."
+
+echo "Running clean-publish --without-publish, as we would before publishing to npm..."
+npx clean-publish --without-publish --before-script ./scripts/clean-shrinkwrap.sh --temp-dir clean
+echo "Packaging cleaned firebase-tools"
+cd ./clean
+PACKED=$(npm pack --pack-destination ../ | tail -n 1)
+cd ..
```

Comment:
Why not leave it in `clean` so it can automatically be cleaned up and removed in one command (above) rather than two? (I'm also guilty of leaving some packed versions in my directory, so I'm a little afraid of your cleanup command anyway)

Reply:
Good call, done